<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPanel</class>
 <widget class="QMainWindow" name="LoginPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>479</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-image: url(:/Image/LoginPanel.png);
}

QLineEdit {

}

QPushButton {

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginPB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameLE">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>282</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLE">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>363</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
